File: app/src/main/java/ru/svolf/pcompiler/util/EditTextPadding.java
package ru.svolf.pcompiler.util;

import android.content.Context;

/**
 * Created by Snow Volf on 07.12.2017, 23:19
 */

public class EditTextPadding {

    public static int getPaddingWithoutLineNumbers(Context context) {
        return (int) PixelDipConverter.INSTANCE.convertDpToPixel(5, context);
    }

    public static int getPaddingBottom(Context context) {
        return (int) PixelDipConverter.INSTANCE.convertDpToPixel(true ? 50 : 0, context);
    }

    public static int getPaddingWithLineNumbers(Context context, float fontSize) {
        return (int) PixelDipConverter.INSTANCE.convertDpToPixel(fontSize * 2f, context);
    }

    public static int getPaddingTop(Context context) {
        return getPaddingWithoutLineNumbers(context);
    }
}

File: app/src/main/java/ru/svolf/pcompiler/util/EditTextPadding.kt
package ru.svolf.pcompiler.util

import android.content.Context
import ru.svolf.pcompiler.util.PixelDipConverter.convertDpToPixel
import ru.svolf.pcompiler.util.PixelDipConverter
import ru.svolf.pcompiler.util.EditTextPadding

/**
 * Created by Snow Volf on 07.12.2017, 23:19
 */
object EditTextPadding {
    fun getPaddingWithoutLineNumbers(context: Context?): Int {
        return convertDpToPixel(5f, context!!).toInt()
    }

    fun getPaddingBottom(context: Context?): Int {
        return convertDpToPixel(50f, context!!).toInt()
    }

    @JvmStatic
    fun getPaddingWithLineNumbers(context: Context?, fontSize: Float): Int {
        return convertDpToPixel(fontSize * 2f, context!!).toInt()
    }

    @JvmStatic
    fun getPaddingTop(context: Context?): Int {
        return getPaddingWithoutLineNumbers(context)
    }
}